common/app/model/content.scala

```diff
@@ -267,6 +267,14 @@ class Content protected (val apiContent: ApiContentWithMeta) extends Trail with
     }
   }
 
+// This is a fix for the hanging letters on long section/series labels on the leftCol breakpoint.
+  lazy val areLabelsLong: Boolean = {
+    val sectionLabelList: Seq[String] = sectionLabelName.split(" ").toSeq
+    val blogLabelList: Seq[String] = blogOrSeriesTag.map(_.name.split(" ").toSeq).getOrElse(Nil)
+    val filteredList: Seq[String] = sectionLabelList.filter(_.length > 10) ++ blogLabelList.filter(_.length > 12)
```

Comment:
This feels very specific and quite brittle. Why can blog labels be longer? Is it in an element with a smaller font size or something?

Either way I'd probably just simplify it to some minimum length after which this behaviour always applies (rather than checking if words within the expression break that length).

The whole function body could then be reduced to

```scala
sectionLabelName.length > 10 || blogOrSeriesTag.exists(_.name.length > 12)
```